cypress/integration/po.types.ts

```diff
@@ -0,0 +1 @@
+export type CypressChainable = Cypress.Chainable;
```

Comment:
We should have the CypressChainable type already included in the @types section of the package.json for the cypress project.

Reply:
I now see why you added in this. Resolving comment.